src/test/java/com/hubspot/dropwizard/guicier/objects/ProvidedProvider.java

```diff
@@ -0,0 +1,7 @@
+package com.hubspot.dropwizard.guicier.objects;
+
+import javax.ws.rs.ext.Provider;
```

Comment:
I think this should be `com.google.inject.Provider`, which means [this dependency](https://github.com/HubSpot/dropwizard-guicier/pull/44/files#r224216380) doesn't need to be added.

Reply:
Yikes, how'd I miss that

Reply:
No, this is correct--it's what InjectedProvider is already using. `Provider` from Guice is the interface with the `get(): T` method